electron/ui/src/views/FlowsheetConfig/FlowsheetConfig.js

```diff
@@ -134,6 +141,9 @@ export default function FlowsheetConfig() {
       setOpenSuccessSaveSnackbar(false);
```

Comment:
I think this is sort of cruft left over from the example the load/save is copied from?

Reply:
This is a little component that pops up briefly when a user saves configuration changes. I think it makes sense to keep this (or something like it) as a quick confirmation that changes were saved.